### DISCUSSION TITLE
v18 beta issues

### DISCUSSION THREAD

**Discussion by kborzecki** [primefaces/community#3121] [PrimeNG]
Hello,
I'm currently working on implementing a custom theme using the new approach available in v18 beta.
I'm noticing a lot of issues and I wonder what is the best place to discuss them.

If possible, I'd like to cooperate with the PrimeTek devs closely (by reporting and providing fixes), without the need of raising GH issues for each problem that I find.

Best regards
Konrad

**Comment by thminuth** [comment]
I am facing the same problems with both v17 and v18 beta. And I doubt that the team will make significant progress with development given the number of problems. In my opinion, the Prime team has taken on too much. The switch to v18, the introduction of the new themes and the focus on Tailwind are just too much. The important core is being neglected. In our team, we have decided not to use Prime any more. We are migrating to NgZorro.

**Reply by kborzecki** [reply]
In my case we've already put too much effort in migrating our current component library to use PrimeNg.
I still strongly believe that PrimeNg was a good choice, but it is true that v18 is far from being ready.
For me the new theming is a good idea as I hope it will be reusable across Angular, Vue and React components. The problem is that some components are not using correct variables, class names or are not using custom templates properly. Those are all rather minor issues, but pretty hard to spot unless you try to write your own, very custom theme.
